src/stylus/fablevision/fv-gene-label.styl

```diff
@@ -0,0 +1,40 @@
+.fv-layout
+
+  .geniblocks.fv-gene-label
+    font-family: 'Yanone Kaffeesatz'
+    font-size: 1.5rem
+    color: white
+    position: absolute
+    display: flex
+    flex-direction: row
+    justify-content: flex-end
+
+    &.m
```

Comment:
This is the biggest problem: at the moment, I'm placing the labels by eye, rather than the gene locations given in the chromosome. I chose this route because the assets I currently have are not based on the BioLogica definitions. It might be worth discussing this a bit more